docker/crypto-detector/consumer/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:22.04
```

Comment:
There are Docker images specifically prepared for Python. Check other examples in our repo like (ignore the rest of contents of the Dockerfile)  https://github.com/giros-dit/semantic-data-aggregator/blob/2c5fcc7df37add1284c0cc0946109a3f5fbce6c7/docker/data-lake-explorer/Dockerfile.

Reply:
Dockerfile now uses python:3.9-slim-buster image